client/src/components/QandA/QandAContainer.jsx

```diff
@@ -105,30 +105,38 @@ function QandA({ product }) {
   };
 
   return (
-    <div className="qaContainer">
-      Questions & Answers
-      <Search filter={setSearch} />
-      {isLoaded
-        ? <p>Loading...</p>
-        : (
-          <QuestionList
+    <div className="qaContainer container">
+      <div className="qaComponent">
+        QUESTIONS & ANSWERS
```

Comment:
I believe the second test job is failing because you changed the text 'Questions & Answers' to all caps while your spec file is testing for lowercase.